docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,292 @@
+Remote Attestation Collaterals
+=====
+
+Adding attestation APIs to support collaterals for remote attestation.  Collaterals are information from a second source that the user/challenger can provide for attestation verification.
```

Comment:
When you say "user/challenger", is that referring to the Relying Party or a Verifier, or what?   (I can't tell what "user" refers to here.)

Reply:
Verifier.  Will add a terminology section at the top.

Reply:
WHy not just use the term Verifier then? :)

Reply:
Done.